samples/Java/Basic/SimpleSend/src/main/java/com/microsoft/azure/eventhubs/samples/SimpleSend/SimpleSend.java

```diff
@@ -20,29 +20,118 @@
 
 public class SimpleSend {
 
-    public static void main(String[] args)
-            throws EventHubException, ExecutionException, InterruptedException, IOException {
+    final URI namespace = new URI("YourEventHubsNamespace.servicebus.windows.net");
+    final String eventhub = "Your event hub";
+    final String authority = "https://login.windows.net/replaceWithTenantIdGuid";
+    final String clientId = "replaceWithClientIdGuid";
+    final String clientSecret = "replaceWithClientSecret";
+    final AuthCallback callback = new AuthCallback(clientId, clientSecret);
 
-        final ConnectionStringBuilder connStr = new ConnectionStringBuilder()
-                .setNamespaceName("Your Event Hubs namespace name") // to target National clouds - use .setEndpoint(URI)
-                .setEventHubName("Your event hub")
-                .setSasKeyName("Your policy name")
-                .setSasKey("Your primary SAS key");
+    public static int main(String[] args)
+        throws EventHubException, ExecutionException, InterruptedException, IOException {
+
+        SimpleSend ss = new SimpleSend();
+        return ss.run(args);
+    }
+
+    private int run(String[] args) throws IOException {
+
+        System.out.println("Choose an action:");
+        System.out.println("[A] Authenticate via Managed Identity and send / receive.");
+        System.out.println("[B] Authenticate via interactive logon and send / receive.");
```

Comment:
B-D are all client secret scenario right now, just three different ways to do the same thing.